### PR TITLE
Password changing functions

### PR DESCRIPTION
**Pull request by IliaStepanov**

### CODE REVIEW

File: src/main/java/com/epam/lowcost/controller/UserController.java [hunk 1]
```diff
@@ -72,19 +72,18 @@ public String updateUser(@RequestParam Map<String, String> params, Model model)
...
+        } else {
             model.addAttribute("user", user);
             model.addAttribute("message", "User successfully updated");
         }
         return "users";
     }
 
     @PostMapping(value = "registration")
-    public String registration(@RequestParam Map<String, String> params,Model model){
+    public String registration(@RequestParam Map<String, String> params, Model model) {
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/controller/UserController.java]: `registration` is a noun

File: src/main/java/com/epam/lowcost/controller/UserController.java [hunk 2]
```diff
@@ -72,19 +72,18 @@ public String updateUser(@RequestParam Map<String, String> params, Model model)
...
-        }
-        else {
+        } else {
             model.addAttribute("user", user);
             model.addAttribute("message", "User successfully updated");
         }
         return "users";
     }
 
     @PostMapping(value = "registration")
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/controller/UserController.java]: Somewhere u use endpoint with `/`, somewhere without. why ?

Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: returns like `return "users"; ` is direct links to .jsp files.

Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: Styled endpoints uniformly.

File: src/main/java/com/epam/lowcost/controller/UserController.java (hunk 1 above)
Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: Changed endpoint to enroll.

Comment (vakhonin) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: no! I highlighted name of method

Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: Changed method name.

**Pull request merged**